pegasus/migrations/114_change_contact_rollups_email_collation.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  # change the contact_rollups email column to utf8_bin collation order. This is so queries from this table sorted by
+  # email get sorted in the exactly the same alphabetization convention as Ruby <=> operator, which is important
+  # for contact sync process.
+  up { run 'ALTER TABLE contact_rollups MODIFY email VARCHAR(255) COLLATE utf8_bin' }
+  down { run 'ALTER TABLE contact_rollups MODIFY email VARCHAR(255) COLLATE utf8_general_ci' }
```

Comment:
It is currently `utf8_general_ci`, I am changing it to `utf8_bin`. These sort differently. (Backtick character has different place in the sort order between the two collations is one difference I've noticed.) Ruby sorts internally using effectively `utf8_bin` so I need to match that.